src/Compute/Compute/help/New-AzVmssConfig.md

```diff
@@ -366,6 +368,36 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -EnableResilientVMCreate
+Enable resilient VM creation.
```

Comment:
Is there anything else we can add here? A link to external docs? what is a resilient vm?

Reply:
Is this the description on the property definition on Swagger too? 
It seems little information

Reply:
This is AI generated but I replaced it with Swagger description.

Reply:
correct

Reply:
somehow the changes didn't get reflected in the help files. Pushed the changes again.